src/include/rbd/librbd.hpp

```diff
@@ -241,6 +241,9 @@ class CEPH_RBD_API Image
   int is_exclusive_lock_owner(bool *is_owner);
   int lock_acquire(rbd_lock_mode_t lock_mode);
   int lock_release();
+  int lock_get_owners(rbd_lock_mode_t *lock_mode,
+                      std::list<std::string> *lock_owners);
+  int lock_break(rbd_lock_mode_t lock_mode, const std::string &lock_owner);
```

Comment:
@dillaman I can't think out a use case for shared lock mode, but if we plan to have this eventually, won't we want to pass several (all) owners to blacklist? Not sure how useful it is -- if the list only partially matches the current owners should we just fail?